easybuild/easyblocks/p/picard.py

```diff
@@ -75,10 +75,20 @@ def sanity_check_step(self):
         jar_files = ['picard']
         if LooseVersion(self.version) < LooseVersion('1.115'):
             jar_files.append('sam')
-        custom_paths = {
-            'files': ["%s-%s.jar" % (x, self.version) for x in jar_files],
-            'dirs': [],
-        }
+        if LooseVersion(self.version) < LooseVersion('1.124'):
+            custom_paths = {
+                'files': ["%s-%s.jar" % (x, self.version) for x in jar_files],
+                'dirs': [],
+            }
+        else:
+            custom_paths = {
+                'files': [
+                    'picard.jar',
+                    'picard-lib.jar',
+                    'htsjdk-%s.jar' % self.version,
```

Comment:
I'm not familiar with Python - I'm a Perl guy! :)

I'll have a bash at it though!